SUPPORT.md

```diff
@@ -0,0 +1,32 @@
+# Commercial Support for CloudNativePG
+
+CloudNativePG is an independent open-source project and does not officially
+endorse any specific company or service provider.
+
+However, to assist users in finding professional support, the
+"[Commercial Support](https://cloudnative-pg.io/support/)"
+page offers an alphabetical list of companies and individuals providing
+CloudNativePG-related products or services.
+
+*Please note that the CloudNativePG authors are not responsible for the accuracy
+or content provided by the listed companies or individuals.*
+
+## How to Get Listed
+
+To have your company or personal services featured on this list, please follow
```

Comment:
maybe mention where one should open the PR? https://github.com/cloudnative-pg/cloudnative-pg.github.io/tree/main/content/support

Reply:
you are right, I had placed a link which was removed in the porting to md

Reply:
is it better now?